.github/workflows/jest.yml

```
@@ -0,0 +1,30 @@
+name: jest
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Is this correct? Seems like it would result in not running on any pull request if it's restricted to main

---

The current config will only run when the PR targets the `main` branch.

If we also want to run against say a `v2` or a`javascript-to-typescript` branch, we need to use a wildcard:

```suggestion
  pull_request:
    branches: [**]
```

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet